### DISCUSSION TITLE
Can Anyone Create a video explaining how we can use searxng and how to make the settings?

### DISCUSSION THREAD

**Discussion by Ahmed-Mohammed-Eid** [open-webui/open-webui#3955] [General]
We need someone help us creating a video explaining how we make the web search works because It's a week now trying to do it and it doesn't work for me, I've already read all the discussions and searched in the web a lot and no thing new the same errors the searxng works fine in docker on "http://localhost:8080" and when I set it in the OpenWebUI it returns an error and I followed the description in the main website hundred times and nothing new, so please help I need it

**Comment by future2233** [comment]
https://docs.searxng.org/admin/engines/settings.html#use-default-settings
Just look at the default settings document provided by the official source. In setting.yml ,  two places that need to be changed:formats add parameter -json ;limiter set to false

**Reply by Ahmed-Mohammed-Eid** [reply]
there's nothing at the link
![Screenshot 2024-07-24 182058](https://github.com/user-attachments/assets/d3852797-b2b9-4ad6-8ace-9e8219eb5be3)

**Reply by justinh-rahb** [reply]
This may be the page they intended to link https://docs.searxng.org/admin/settings/settings.html#use-default-settings

**Comment by delfireinoso** [comment]
In Adminstration Panel
Web search 
http://host.docker.internal:8000/search?q=<query>
